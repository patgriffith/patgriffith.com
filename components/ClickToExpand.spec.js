import { shallowMount } from '@vue/test-utils';
import ClickToExpand from './ClickToExpand';

describe('<ClickToExpand />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(ClickToExpand);
  })

  it('should render <ClickToExpand/> without crashing', () => {
    expect(wrapper.element).toMatchSnapshot();
  })
})
