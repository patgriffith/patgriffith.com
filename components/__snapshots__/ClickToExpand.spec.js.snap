// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClickToExpand /> should render <ClickToExpand/> without crashing 1`] = `
<div
  class="mx-auto max-w-xl"
>
  <div
    class="max-w-md text-right mr-20 md:mr-0"
  >
    <img
      alt="Click to expand."
      class="inline-block mr-12 mt-4"
      height="49"
      src="/img/click-to-expand.png"
      width="212"
    />
  </div>
</div>
`;
